test/FigoTest.php

```diff
@@ -29,210 +29,174 @@
 
 class SessionTest extends PHPUnit_Framework_TestCase {
 
-    protected $sut;
+    protected static $api_endpoint = "https://staging.figo.me/v3";
+    protected static $fingerprints = array("D0:03:9E:F0:8F:BD:48:67:86:71:CE:9D:A5:54:24:81:63:D7:D9:4D:ED:F1:6A:55:F0:52:C7:0A:AB:7B:B8:9D");
 
-    protected function setUp() {
-        $this->sut = new Session("ASHWLIkouP2O6_bgA2wWReRhletgWKHYjLqDaqb0LFfamim9RjexTo22ujRIP_cjLiRiSyQXyt2kM1eXU2XLFZQ0Hro15HikJQT_eNeT_9XQ");
+    protected static $connection;
+    protected static $email;
+    protected static $password;
+    protected static $session;
+
+    protected $access_token;
+    protected $account_id;
+    protected $sec_account_id;
+
+    public static function setUpBeforeClass()
+    {
+        self::$connection = new Connection(getenv("FIGO_CLIENT_ID"), getenv("FIGO_CLIENT_SECRET"),
+            "http://my-domain.org/redirect-url", self::$api_endpoint, self::$fingerprints);
+        $name = "PHP SDK Test";
+        self::$email = "php.sdk.".rand()."@figo.io";
+        self::$password = "sdk_test_pass_".rand();
+        self::$connection->create_user($name, self::$email, self::$password);
+        $response = self::$connection->native_client_login(self::$email, self::$password);
+        $access_token = $response["access_token"];
+        self::$session = new Session($access_token, self::$api_endpoint, self::$fingerprints);
     }
 
-    public function test_accounts() {
-        $accounts = $this->sut->get_accounts();
-        $this->assertGreaterThan(0, count($accounts));
+    public static function tearDownAfterClass()
+    {
```

Comment:
Please unify the way you use curly braces. Currently we have different styles here:

```php
public static function tearDownAfterClass() {
    ...
```
vs
```php
public static function tearDownAfterClass() 
{
    ...
```